modules/sdk/src/main/scala/org/tessellation/sdk/package.scala

```diff
@@ -45,6 +46,12 @@ package object sdk {
       TimedOut.getClass -> 519,
       classOf[PeerMismatch] -> 520,
       classOf[PeerCheckTimeouted] -> 521,
-      classOf[PeerCheckUnexpectedError] -> 522
+      classOf[PeerCheckUnexpectedError] -> 522,
+      classOf[Host] -> 523,
```

Comment:
I will create a ticket for figuring out if we can somehow force Kryo to use our logger. We spend so much time because of these errors.